.env.example

```
@@ -1,3 +1,6 @@
 # Database Configuration
 # Devbox default postgresql connection string
-DATABASE_URL="postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable"
+DATABASE_URL="postgres://user:password@host:port/dbname"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)